pkg/roachprod/promhelperclient/client.go

```diff
@@ -67,7 +67,8 @@ func NewPromClient() *PromClient {
 // instanceConfigRequest is the HTTP request received for generating instance config
 type instanceConfigRequest struct {
 	//Config is the content of the yaml file
-	Config string `json:"config"`
+	Config   string `json:"config"`
+	Insecure bool   `json:"insecure"`
```

Comment:
Could you explain to me how prom knows to make an insecure request based off of this? I see in the [prom helper](https://github.com/cockroachlabs/prom-helper-service/blob/db9f7ccb5fac0d681abb61aa956e56509e68ae96/handlers/instanceconfigs/put.go#L68-L71), we append insecure to the config file name:

```
	if r.Insecure {
		configDirSuffix = inSecureConfigDirSuffix
	}
	fileName := fmt.Sprintf("%s%s/%s.%s", instanceConfigDir, configDirSuffix,
```

Is that all prom needs to know? It doesn't look like we do anything else with `r.Insecure`.

Reply:
That is correct @DarrylWong . There are  locations where the target files are created: `instance-configs` and `instance-configs-insecure`

Reply:
Ahh, and then prom has two different jobs based on the directory? That makes sense, although a tiny nit would be to maybe leave a one line comment explaining that. Otherwise you'd have to dig through the prometheus.yml file to find that info.

Reply:
THanks @DarrylWong ! I added the comment in the promhelpers repo.